cilium-cli/connectivity/check/features.go

```diff
@@ -323,40 +323,46 @@ func (ct *ConnectivityTest) detectFeatures(ctx context.Context) error {
 	if cm.Data == nil {
 		return fmt.Errorf("ConfigMap %q does not contain any configuration", defaults.ConfigMapName)
 	}
+
```

Comment:
nit: I'd add in the release note `cilium-cli: Optimized feature detection by running cluster-wide checks once.`